plugins/nanospell/plugin.js

```diff
@@ -313,28 +324,33 @@
 
 			/* #2 setup layer */
 			/* #3 nanospell util layer */
-			var start = function () {
+			function start() {
 				editor.getCommand('nanospell').setState(CKEDITOR.TRISTATE_ON);
 				state = true;
-				var words = getWords(editor.document.$.body, maxRequest);
```

Comment:
this stuff was actually duplicate, kinda.  and the max length stuff wasn't really working